src/main/java/sirius/biz/web/SaveHelper.java

```diff
@@ -67,6 +68,25 @@ public SaveHelper withPostSaveHandler(Consumer<Boolean> postSaveHandler) {
         return this;
     }
 
+    /**
+     * Installs a predicate which can inspect the underlying entity and determine if a save for a POST should be
+     * attempted or not.
+     * <p>
+     * Note that these predicates are a bit exotic and only used if an entity is to be created in a two-step
+     * process. These are entities where some fields need to be set (like a type) before other fields
+     * can even be rendered (e.g. if their type or appearance depends on the type itself).
+     *
+     * @param skipCreatePredicate the predicate which decides if the entity should be persisted in the database
+     *                            or if the editor should be rendered again. They receive the {@link BaseEntity#isNew()}
+     *                            as if was before loading any data and may return <tt>true</tt> to save the entity
+     *                            or <tt>false</tt> to show the editor again.
+     * @return the helper itself for fluent method calls
+     */
+    public SaveHelper withSkipCreatePredicate(UnaryOperator<Boolean> skipCreatePredicate) {
```

Comment:
shouldnt this actually be a Predicate Entity -> boolean as also hinted at by the name? how can the predicate inspect the entity if it only recieves BaseEntity#isNew , shouldnt they receive the whole entity ?

Reply:
the entity is normally visible to the predicate / closure passed in. we only forward the isNew flag, as we capture it before any changes are performed (this is the same for all other handlers...)